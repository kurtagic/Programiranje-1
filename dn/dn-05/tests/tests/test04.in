15
1 89
1 90
0 87
2 86
2 84
2 91
0 87
1 87
0 84
1 88
1 91
0 87
2 84
1 88
1 85
