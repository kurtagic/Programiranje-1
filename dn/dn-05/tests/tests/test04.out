84: 15
85: 19
86: 13
87: 14
88: 18
89: 14
90: 14
91: 6
92: 2
